pyezviz/__main__.py

```diff
@@ -124,7 +124,7 @@ def main():
 
         # load camera object
         try:
-            client.login()
+            client._login()
             camera = EzvizCamera(client, args.serial)
```

Comment:
Missing positional argument camerainfo

Reply:
Hi There!

The EzvizCamera class needs to be called with an additional variable "camerainfo" 

EzvizCamera(self, device['deviceSerial'], camerainfo)

The json array "camerainfo" is created with the "_get_deviceinfo" function in the client.py file. I've done it this way as to reduce api calls. 

I've only tested with devices status. I see there are other calls to the EzvizCamera class, in the __Main__.py file that i missed . I'll work on fixing them.

I'm new to python so please excuse the minor errors. I have +- 2 weeks of experience so far but the basics seem simple enough.

Reply:
Should work now.

Reply:
> Missing positional argument camerainfo

Following the instructions on using this with HA. I am seeing the same issue 
I believe the component has been loaded successfully with the correct version "Requirement already satisfied: **pyezviz** in /usr/local/lib/python3.8/site-packages **(0.1.5.9**)"

`2020-12-20 12:36:31 ERROR (MainThread) [homeassistant.components.camera] Error while setting up ezviz_my platform for camera
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 199, in _async_setup_platform
    await asyncio.shield(task)
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/ezviz_my/camera.py", line 91, in setup_platform
    camera["ezviz_camera"] = EzvizCamera(ezviz_client, camera_serial)
TypeError: __init__() missing 1 required positional argument: 'camerainfo'`

Using the EZVIZ C6N camera.

Reply:
Got it working by adding
`        cameras = ezviz_client.load_cameras()`

after line 51 in HA EZVIZ component (camera.py)
```
    try:
        ezviz_client = EzvizClient(account, password)
        ezviz_client.login()
        cameras = ezviz_client.load_cameras()
```

And changing former line 91 (now 92) from
`        camera["ezviz_camera"] = EzvizCamera(ezviz_client, camera_serial)`
to
`        camera["ezviz_camera"] = EzvizCamera(ezviz_client, camera_serial, camerainfo)`

Reply:
I think RenierM26 already fixed those issues in the latest revision. I only tested the command-line though, have not integrated it with Home Assistant.

Reply:
Isn't 0.1.5.9 the latest version via master branch?
`pip install git+https://github.com/RenierM26/pyEzviz.git@master#pyezviz==0.1.5.6` is the version of the pyezviz I install.
(git is not my area of expertise).

Reply:
That's cool, I didn't know you could install packages from git with pip.
I tried this command in a clean environment and it installed pyEzviz 0.1.5.9 with RenierM26's fixes already applied:
```
pip install git+https://github.com/RenierM26/pyEzviz.git@master
```

Reply:
Just found that out myself looking into this stuff. 
pyEzviz 0.1.5.9 in itself works just fine for me as well.

But Home Assistant EZVIZ component has the following line of code: `camera["ezviz_camera"] = EzvizCamera(ezviz_client, camera_serial)`. Which errors out. As RenierM26s pyEzviz 0.1.5.9 EzvizCamera require a third argument.
Ref:
```
class EzvizCamera(object):
    def __init__(self, client, serial, camerainfo):
        """Initialize the camera object."""
        self._client = client
        self._serial = serial
        self._camerainfo = camerainfo
```

Reply:
Oh I see, the Home Assistant component is missing something. Sorry I misunderstood the problem.
https://github.com/home-assistant/core/blob/dev/homeassistant/components/ezviz/camera.py#L91